Sources/swift-algorand-sdk/v2/client/IndexerClient.swift

```diff
@@ -11,7 +11,7 @@ public class IndexerClient {
     var host:String
     var port:String
     var token:String
-    var apiKey="X-Algo-API-Token"//X-API-Key
+    var apiKey="x-api-key"//X-API-Key
```

Comment:
use the method `func set(key:String)` to change your API key. You are suggesting a breaking change for everyone else.